trino/dbapi.py

```diff
@@ -465,7 +465,28 @@ def execute(self, operation, params=None):
         return result
 
     def executemany(self, operation, seq_of_params):
-        raise trino.exceptions.NotSupportedError
+        """
+        PEP-0249: Prepare a database operation (query or command) and then
+        execute it against all parameter sequences or mappings found in the sequence seq_of_parameters.
+        Modules are free to implement this method using multiple calls to
+        the .execute() method or by using array operations to have the
+        database process the sequence as a whole in one call.
+
+        Use of this method for an operation which produces one or more result
+        sets constitutes undefined behavior, and the implementation is permitted (but not required)
+        to raise an exception when it detects that a result set has been created by an invocation of the operation.
+
+        The same comments as for .execute() also apply accordingly to this method.
+
+        Return values are not defined.
+        """
+        for parameters in seq_of_params[:-1]:
+            self.execute(operation, parameters)
+            self.fetchall()
```

Comment:
Can we identify and throw if a result set was returned? To prevent people from using this with SELECT queries and then wondering where the results go.

Reply:
I am going to use updateType field, so multiple INSERT should be fine.

Reply:
We can adjust this condition over time. updateType looks good to me.